actioncable/lib/rails/generators/channel/channel_generator.rb

```diff
@@ -9,14 +9,17 @@ class ChannelGenerator < NamedBase
 
       check_class_collision suffix: "Channel"
 
+      hook_for :javascript_engine
```

Comment:
Ah yeah, this attempts to run a `coffee:generators:channel` (or `js:`) command 😁